### PR TITLE
Update urls.md

### PR DESCRIPTION
**Pull request by falyne**
Capitalized the url
Added new line for easy browsing

### CODE REVIEW

**Comment by minrk** [comment]
This PR now has conflicts because of an earlier contribution on the same file. Please resolve the conflicts and update the pull request. If you're not comfortable with resolving conflicts, you can make fresh changes and open a new PR, linking to this one and closing this one.

**Comment by falyne** [comment]
Hi @minrk Thanks for reviewing.
I have resolved the the Conflicts as adviced earlier

File: docs/source/reference/urls.md [hunk 1]
```diff
@@ -120,8 +120,8 @@ if the specified server were running).
 Handling this URL is the most complicated condition in JupyterHub,
 because there can be many states:
 
-1. the server is not active
-   a. user matches
+1. the server is not active/
```
Comment (minrk) [inline docs/source/reference/urls.md:123]: Can you clarify why have these slashes been added? I don't understand what they are meant to communicate.

**Comment by minrk** [comment]
Thanks for your contribution! I'm going to close this PR, as adding slashes to the ends of lists isn't a change we should make.

**Pull request closed without merging**